nasa-gibs/worldview
# WV-3115: Updates for surface water extent story

## Description

Fixes #WV-3115.

More updates to the OPERA Surface Water Extent Tour Story. Still waiting on some issues to be resolved for the imagery in step 7. 

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-174.25840338686484,-87.17983616504753,141.53238355885279,88.5845925717814&l=Coastlines_15m,Reference_Features_15m,Reference_Labels_15m,OPERA_L3_Dynamic_Surface_Water_Extent-HLS_Provisional(disabled=5),Land_Water_Map&lg=true&tr=surface_water_extent&t=2023-09-16-T00%3A00%3A00Z)
2. Go through tour story and make sure steps work.

@nasa-gibs/worldview
